package.json

```diff
@@ -4,9 +4,11 @@
   "description": "Constants and utilities about visitor keys to traverse AST.",
   "type": "module",
   "main": "dist/eslint-visitor-keys.cjs",
+  "types": "./dist/index.d.ts",
   "exports": {
     ".": [
       {
+        "types": "./dist/index.d.ts",
```

Comment:
Just checking: do we need this in two places?

Reply:
No, I've removed it. Per https://www.typescriptlang.org/docs/handbook/esm-node.html#packagejson-exports-imports-and-self-referencing , the other `types` is the fallback for older versions of TypeScript so probably best to keep that one.

Such `types` within `exports` might be usable if we were to export the `visitor-keys` file separately, as that has its own declaration file, but as we're not doing that, it's not necessary, so removed.